airbyte-integrations/connectors/source-trustpilot/integration_tests/configured_catalog.json

```diff
@@ -0,0 +1,97 @@
+{
+  "streams" : [
+    {
+      "stream" : {
+        "name" : "reviews",
+        "json_schema" : {
+          "$schema" : "http://json-schema.org/draft-07/schema#",
+          "title" : "Trustpilot reviews",
+          "description" : "Trustpilot reviews schema",
+          "type" : "object",
+          "required" : [
+            "@type",
+            "@id",
+            "datePublished"
+          ],
+          "properties" : {
+            "@type" : {
+              "const" : "Review"
+            },
+            "@id" : {
+              "type" : "string"
+            },
+            "itemReviewed" : {
+              "type" : "object",
```

Comment:
You don't need to add the schema here, you can change by `"json_schema": {}`

Reply:
Thank You for advice!

Reply:
Fixed.